Objects/object.c

```diff
@@ -1172,7 +1172,13 @@ PyObject_SetAttr(PyObject *v, PyObject *name, PyObject *value)
 
     PyUnicode_InternInPlace(&name);
     if (tp->tp_setattro != NULL) {
-        err = (*tp->tp_setattro)(v, name, value);
+        if(Py_CHECKWRITE(v)){
+            err = (*tp->tp_setattro)(v, name, value);
+        }else{
```

Comment:
```suggestion
        }
        else{
```

Reply:
Is this just a style thing? I think the cpython style seems to be open brace on same line. They have some linting they run as a precommit hook and I'm passing that atm.